.rubocop_todo.yml

```diff
@@ -130,13 +130,13 @@ RSpec/StubbedMock:
 # Cop supports --auto-correct.
 # Configuration parameters: Include.
 # Include: config/routes.rb, config/routes/**/*.rb
-Rails/MatchRoute:
-  Exclude:
-    - 'modules/appeals_api/config/routes.rb'
-    - 'modules/apps_api/config/routes.rb'
-    - 'modules/claims_api/config/routes.rb'
-    - 'modules/va_forms/config/routes.rb'
-    - 'modules/vba_documents/config/routes.rb'
+# Rails/MatchRoute:
```

Comment:
Instead of commenting out, let's remove.

Reply:
Oops, forgot to remove that. Just pushed a fix.